src/gsi/cplr_get_fv3_regional_ensperts.f90

```diff
@@ -447,7 +477,7 @@ subroutine general_read_fv3_regional(this,fv3_filenameginput,g_ps,g_u,g_v,g_tv,g
                 end do
              end do
            end do
-           call genqsat(g_rh,g_tsen(1,1,1),g_prsl(1,1,1),grd_ens%lat2,grd_ens%lon2,grd_ens%nsig,ice,iderivative)
+           call genqsat(g_rh,g_tsen(1,1,1),g_prsl(1,1,1),lat2ens,lon2ens,grd_ens%nsig,ice,iderivative)
```

Comment:
Hi, Bin, 
Can you replace all the lat2ens,lon2ens with grd_ens%lat2,grd_ens%lon2?  And remove the definition of lat2ens/lon2ens at L334 & L365.   
Or can you get a new cplr_get_fv3_regional_ensperts.f90 from /work/noaa/aoml-hafsda/xlu/HAFS_V16/sorc/hafs_gsi.fd/src/gsi/?  There is no need to introduce extra variables when the lat2ens=grd_ens%lat2

Reply:
Thanks @XL-OU! Copied your file and committed into this branch.